cg/meta/upload/mip/mip_rna.py

```diff
@@ -34,14 +34,16 @@ def upload(self, ctx: click.Context, case: Family, restart: bool) -> None:
 
         # Scout specific upload
         if DataDelivery.SCOUT in case.data_delivery:
-            ctx.invoke(upload_rna_to_scout, case_id=case.internal_id)
+            result: int = ctx.invoke(upload_rna_to_scout, case_id=case.internal_id)
+            if result == 0:
+                LOG.info(
+                    f"Upload of case {case.internal_id} was successful. Setting uploaded at to {dt.datetime.now()}"
+                )
+                self.update_uploaded_at(analysis_obj)
+            else:
+                raise RuntimeError(f"Upload to Scout failed for sample {case.internal_id}")
```

Comment:
Can you make sure that the logging is clear when this happens? `Upload to Scout failed for sample xxxx` is quite vague, but if the previous error messages are shown, it's fine 👍

Reply:
This is what get now:
```bash
----------------- UPLOAD RNA FUSION REPORT TO SCOUT -----------------------
Failed to upload files for RNA case: unexpected number of DNA sample matches for subject_id: GMSB-UM038. Number of matches: 2 
Traceback (most recent call last):
  File "/home/proj/stage/bin/miniconda3/envs/S_main/bin/cg", line 8, in <module>
    sys.exit(base())
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/core.py", line 1043, in invoke
    return Command.invoke(self, ctx)
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/cg/cli/upload/base.py", line 82, in upload
    upload_api.upload(ctx=context, case=case, restart=restart)
  File "/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/cg/meta/upload/mip/mip_rna.py", line 44, in upload
    raise RuntimeError(f"Upload to Scout failed for sample {case.internal_id}")
RuntimeError: Upload to Scout failed for sample closefawn
```